client/src/templates/Challenges/components/challenge-title.css

```diff
@@ -0,0 +1,17 @@
+.challenge-title-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 1.45rem;
+}
+
+.challenge-title {
+  display: inline;
+  padding: 0 15px;
+  margin: 0;
+  flex-grow: 2;
+}
+
+.challenge-title-wrap > Button {
```

Comment:
```suggestion
```
It looks a bit weird when the buttons resize, so I'd recommend leaving them fixed.